programmers/4일차/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,36 @@
+function solution(lottos,winNums){
+    const correctCntRank = {
+        6: 1,
+        5: 2,
+        4: 3,
+        3: 4,
+        2: 5,
+        1: 6,
+        0: 6
+    }
+    let minCount = 0;
+    let zeroCount = 0;
+    let maxCount = 0;
+    //minCount,maxCount 은 로또의 최소,최고 등수일때 zeroCount 은 lottos에 있는 0의 개수
+    
+    for (let zero of lottos) {
+        if (zero == 0) zeroCount += 1;
```

Comment:
```suggestion
        if (zero === 0) zeroCount += 1;
```
아 이거 전에 문제에서는 === 연산자를 사용하셔서  피드백 안드렸는데  비교 연산자는 무조건  == 사용은 지양해주시고 === 연산자를 사용해주시는게 바람직합니다  ==는  동등연산자이고 ===는 일치 연산자라고 하는데
이것도 한번 찾아서 공부해보시는걸 추천드릴게요 글로 설명하려고 하니 너무 길어져서  == 는  정수 1과  문자열 '1' 도 같다고 간주를 해요  자료형 까지는 비교하는데 있어서 반형을 하지 않는 거죠  반면 === 연산자는  자료형까지도 일치하는지 비교를 해줍니다  따라서 코드 품질에 있어서 일치 연산자를 사용하는게  바람직합니다 참고해주시면 좋을거 같아요